### ISSUE TITLE
Layout for "2 months in 2 minutes"?

### ISSUE THREAD

**Issue by maelle** [ropensci/roweb2#679]
Something to keep in mind for the redesign, we might want special layouts for such posts.

The layout name would be indicated in posts' frontmatter. https://gohugo.io/templates/lookup-order/

**Comment by maelle** [comment]
@stefaniebutland this is the issue where to write whether you'd like to enter content for "2 months in 2 minutes" in a different way (e.g. for part of it using YAML instead of writing stuff). :wink:

And also if any tool could help you get the content more easily, of course.

**Comment by stefaniebutland** [comment]
Agreed we'll discuss when new website is settled. Make it look nicer. 

Also Scott, Stef & Maëlle can figure out how to make it faster using scripts to copy some newsletter content to 2mon2min template
